src/applications/personalization/profile/mocks/server.js

```diff
@@ -129,7 +135,8 @@ const responses = {
     }
     // return res.status(403).json(genericErrors.error500);
     // example user data cases
-    return res.json(user.loa3User72); // default user LOA3 w/id.me (success)
+    return res.json(user.loa3UserNeedsVapInit);
```

Comment:
I'd rather see this commented line preserved for easy testing in the future.

```suggestion
    // return res.json(user.loa3User72); // default user LOA3 w/id.me (success)
    return res.json(user.loa3UserNeedsVapInit);
```